Make it easier to understand how to setup default selected and checked items for newbie's (#434)
octobercms/docs: backend-forms.md

**comment** | places a descriptive comment below the field.
**commentAbove** | places a comment above the field.
**commentHtml** | allow HTML markup inside the comment. Options: true, false.

> **default** | specify the `value` field in the options list you want to make selected or checked on page load, compatible with `dropdown`, `checkboxlist`, `radio` and `balloon-selector` widgets.

Reviewer: @ayumi-cloud 
```suggestion
**default** | specify the default value for the field. For `dropdown`, `checkboxlist`, `radio` and `balloon-selector` widgets, you may specify an option key here to have it selected by default.
```
